### PR TITLE
Playwright: Ensure that master key is the expected one

### PR DESCRIPTION
**Pull request by BillCarsonFr**
Quick improvements to the playwright verification test:
 - Ensure that the new session sees the correct cross signing key (protects against regression that would rotate the key on the new session)
 - Check also that the backup decryption key is correctly gossiped after interactive verification (only rust crypto as on legacy it's a bit flaky because the setting page doesn't refresh when secret is received; would have to close and re-open or wait before opening first time)

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

File: playwright/e2e/crypto/utils.ts [hunk 1]
```diff
@@ -14,13 +14,13 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import { type Page, expect, JSHandle } from "@playwright/test";
+import { expect, JSHandle, type Page } from "@playwright/test";
 
-import type { CryptoEvent, ICreateRoomOpts, MatrixClient } from "matrix-js-sdk/src/matrix";
+import type { ICreateRoomOpts, MatrixClient } from "matrix-js-sdk/src/matrix";
```
Comment (richvdh) [inline playwright/e2e/crypto/utils.ts:19]: `CryptoEvent` is still referenced below, so this needs to stay, I think.

Note that it's important it be `import type` rather than `import` (since playwright tests aren't allowed to have runtime dependencies on the js-sdk).

File: src/components/views/settings/SecureBackupPanel.tsx [hunk 2]
```diff
@@ -96,6 +96,7 @@ export default class SecureBackupPanel extends React.PureComponent<{}, IState> {
 
         MatrixClientPeg.safeGet().on(CryptoEvent.KeyBackupStatus, this.onKeyBackupStatus);
         MatrixClientPeg.safeGet().on(CryptoEvent.KeyBackupSessionsRemaining, this.onKeyBackupSessionsRemaining);
+        MatrixClientPeg.safeGet().on(CryptoEvent.KeyBackupDecryptionKeyCached, this.onKeyBackupDecryptionKeyCached);
```
Comment (richvdh) [inline src/components/views/settings/SecureBackupPanel.tsx:99]: this looks like a functionality change. Please could you either pull it out to a separate PR, or update this PR to emphasise that this is a functionality change with some tests thrown in?

**Review by richvdh — CHANGES_REQUESTED** [review]

**Comment by richvdh** [comment]
@BillCarsonFr are you still working on this? it seems to be bitrotting.

**Comment by t3chguy** [comment]
Closing due to being abandoned
